test/test_block_genesis.py

```diff
@@ -0,0 +1,55 @@
+"""
+Test genesis block
+"""
+
+import pytest
+
+from .util import get_block
+
+FEE_TOKEN_CLASS_HASH = "0x6a22bf63c7bc07effa39a25dfbd21523d211db0100a0afd054d172b81840eaf"
+UDC_CLASS_HASH = "0x7b3e05f48f0c69e4a65ce5e076a66271a527aff2c34ce1083ec6e1526997a69"
+CHARGEABLE_ACCOUNT_CLASS_HASH = "0x4d07e40e93398ed3c76981e72dd1fd22557a78ce36c0515f679e27f0bb5bc5f"
+STARKNET_CLI_ACCOUN_CLASS_HASH = "0x6f500f527355dfdb8093c7fe46e6f73c96a867392b49fa4157a757538928539"
```

Comment:
typo `ACCOUN`

Reply:
fixed in 9132fcf75d839abcea5bf652d11a60944885a190